<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkDialog</class>
 <widget class="QDialog" name="LinkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Link Text:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="urlLink">
     <property name="placeholderText">
      <string>http://NotePages.org/</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="linkText"/>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="pageLink"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Page Link:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>pageBttn</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Website/URL:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>urlBttn</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QRadioButton" name="pageBttn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QRadioButton" name="urlBttn">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>linkText</tabstop>
  <tabstop>urlBttn</tabstop>
  <tabstop>pageBttn</tabstop>
  <tabstop>urlLink</tabstop>
  <tabstop>pageLink</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LinkDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LinkDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>urlLink</sender>
   <signal>textEdited(QString)</signal>
   <receiver>urlBttn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pageLink</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>pageBttn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
